plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -115,6 +126,23 @@ def create_optuna_distribution_from_override(override: Override) -> Any:
     raise NotImplementedError(f"{override} is not supported by Optuna sweeper.")
 
 
+def create_params_from_overrides(
+    arguments: List[str],
+) -> Tuple[Dict[str, BaseDistribution], Dict[str, Any]]:
+    parser = OverridesParser.create()
+    parsed = parser.parse_overrides(arguments)
+    search_space = dict()
```

Comment:
In `OptunaSweeperImpl.__init__`, the `search_space` variable has type `Optional[DictConfig]`, but in this function `search_space` has type `Dict[str, BaseDistribution]`. This type mismatch is confusing... Could you please rename `search_space` here to a different name?

Reply:
Changed it to search_space_distributions